daml-lf/spec/transaction.rst

```diff
@@ -502,9 +502,9 @@ As version 12, these field are included:
 ``arg_unversioned`` and ``result_unversioned`` are required, while
 ``arg_versioned`` and ``result_versioned`` are not used anymore.
 
-(*since version 1.14*)
+(*since version 14*)
 
-As of version 1.dev, this field is required:
+As of version 14, this field is required:
```

Comment:
I found it a bit curious that this file sometimes uses "version 1.x" and sometimes "version x", I think we should pick one (not for this PR though)

Reply:
Yes, this puzzled me too.

Reply:
transaction version are not LF version (see [TransactionVersion.scala](https://github.com/digital-asset/daml/blob/4aa5b7d7685895399aa231917aac5875fe5a46f1/daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/TransactionVersion.scala#L57) 
```
  private[lf] val assignNodeVersion: LanguageVersion => TransactionVersion = {
    import LanguageVersion._
    Map(
      v1_6 -> V10,
      v1_7 -> V10,
      v1_8 -> V10,
      v1_11 -> V11,
      v1_12 -> V12,
      v1_13 -> V13,
      v1_14 -> V14,
      v1_15 -> V15,
      v1_dev -> VDev,
    )
```

starting from LF 1.11, we decide to bind them, but still they are not the same think.
If we are able to drop old LF version, we will be able to make those concept the same.